.circleci/continue-config.yml

```
@@ -135,13 +153,22 @@ workflows:
           context: dev-test
           release_engine: 'lite'
           commit_message: 'chore(release): publish --skip-ci'
+
           requires:
             - test
             - e2e
           filters:
             branches:
               only: master
 
+      - publish-to-chromatic:
+          context: dev-test
+          requires:
+            - vfcommon/monorepo_release
```

I think we should just require `test` so that this runs at the same time as `monorepo_release`
```suggestion
            - test
```